_data/chains/eip155-7887.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "Kinto Mainnet",
+    "chain": "Kinto Mainnet",
+    "rpc": [
+      "https://rpc.kinto.xyz/http",
+      "https://kinto-mainnet.calderachain.xyz/http"
+    ],
+    "nativeCurrency": {
+      "name": "Ethereum",
+      "symbol": "ETH",
+      "decimals": 18
+    },
+    "infoURL": "https://kinto.xyz/",
+    "shortName": "kintoMainnet",
+    "chainId": 7887,
+    "networkId": 7887,
+    "icon": "kinto",
+    "explorers": [
+      {
+        "name": "Kinto Explorer",
+        "url": "https://explorer.kinto.xyz/",
+        "icon": "kinto",
```

Comment:
do you need an url here?

Reply:
the icon string references to this path: [_data/icons/kinto.json](https://github.com/ConstellationCrypto/chains/pull/1/files#diff-6fd79ccdd392c39d04ffbdad9d19b9299a62d048a30cfe49247613e35d889465)

I took the icon from your twitter, let me know if there's a different one I should use. It needs to be approximately square

Reply:
perfect